taxonomy/migrations/0033_auto_20230614_1246.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.18 on 2023-06-14 12:46
```

Comment:
Can we update the name of the auto-generated migration to represent the database changes? Maybe something like... `0033_add_b2c_job_allowlist`?

Reply:
Changed. it's now 0033_b2c_allowlist.py